src/site/_data/postTags.js

```diff
@@ -189,10 +189,17 @@ const tags = {
 
 const postTags = {};
 
-Object.keys(tags).forEach((key) => {
-  postTags[key] = {
-    ...tags[key],
-    href: `/tags/${key}/`,
+Object.keys(tags).forEach(($key) => {
```

Comment:
Why `$key`? I know it's a valid JS variable name, just...  it seems like a random change.